test/Interop/CxxToSwiftToCxx/span/span-execution.cpp

```diff
@@ -21,6 +21,10 @@
 using Span = std::span<int>;
 using SpanOfString = std::span<std::string>;
 
+namespace Cxx {
```

Comment:
This naming is kind of asking for trouble 😅
Swift might confuse it with the `Cxx` module, which is a part of the stdlib overlay.

Reply:
Makes sense, thanks!